Source/SpectrumAnalysis/KTSequentialTrackFinder.hh

```diff
@@ -202,8 +213,10 @@ namespace Katydid
             //***************
 
         private:
-            Nymph::KTSlotDataThreeTypes < KTSliceHeader, KTPowerSpectrumData, KTDiscriminatedPoints1DData > fDiscrimPowerSlot;
-            Nymph::KTSlotDataTwoTypes < KTSliceHeader, KTDiscriminatedPoints1DData > fDiscrimSlot;
+            Nymph::KTSlotDataOneType< KTEggHeader > fHeaderSlot;
+            Nymph::KTSlotDataThreeTypes< KTSliceHeader, KTPowerSpectrumData, KTDiscriminatedPoints1DData > fDiscrimPowerSlot;
+            Nymph::KTSlotDataTwoTypes< KTSliceHeader, KTDiscriminatedPoints1DData > fDiscrimSlot;
+            Nymph::KTSlotDataOneType< KTKDTreeData > fDiscrimKDTreeSlot;
             Nymph::KTSlotDone fDoneSlot;
 
```

Comment:
That was a lot of STF cleaning :+1: Thank you!

Reply:
You're welcome :-)